src/drawDirective.coffee

```diff
@@ -25,12 +25,13 @@ angular.module('ui-leaflet')
     scope: false
     replace: false
     require: ['leaflet']
-    controller: ($scope) ->
+    controller: ['$scope', ($scope) ->
```

Comment:
You're running version `2` of `gulp-ng-annotate` (https://github.com/angular-ui/ui-leaflet-draw/blob/master/package.json#L34) and that's the newest: https://github.com/Kagami/gulp-ng-annotate/releases. It seems we can't update that gulp plugin.

I'm reading your comment as "update the ngAnnotate version for ui-leaflet-draw because that might produce the final .js file with the array value for the controller that we want" but as per above, we can't. If I've got it wrong and you're recommending that I update ngAnnotate in my project that uses ui-leaflet-draw (not ui-leaflet-draw itself); that's a valid request and I can give it a go. **Which interpretation is correct?**

It was a while ago when I did this stuff so I had to get back into what I was doing. I was lazy and was using my project at the time to test changes I made to this module. I was trying to minify that project and clearly I thought this line in ui-leaflet-draw was causing an issue. Unfortunately that makes it hard to reproduce the problem now because I've lost that test harness (lesson learnt).

Disclaimer:
I'm still pretty new to the NodeJS and Angular world so if it seems like I'm doing something stupid, I probably am :confounded: .